### PR TITLE
Add Android UIComponents example

### PR DESCRIPTION
**Pull request by lslachciak**
Added new UIComponents Android example.
Bases on the default simple Android project

### CODE REVIEW

File: examples/Android/UIComponents/app/src/main/java/com/example/uicomponents/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+    package com.example.uicomponents
```
Comment (HunseopJeong) [inline examples/Android/UIComponents/app/src/main/java/com/example/uicomponents/MainActivity.kt]: Need to remove whitespace.

**Review by TomaszLukawskiSam — APPROVED** [review]
I opened, built and install the app on android device without any problems.

Except for the issue found by @HunseopJeong,
I approve this PR.

File: examples/Android/UIComponents/app/src/main/java/com/example/uicomponents/MainActivity.kt (hunk 1 above)
Comment (lslachciak) [reply examples/Android/UIComponents/app/src/main/java/com/example/uicomponents/MainActivity.kt]: done

**Comment by lslachciak** [comment]
Improved PR according to the @HunseopJeong comment

**Review by TomaszLukawskiSam — APPROVED** [review]

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**